apps/cloud_inference/server_app/CMakeLists.txt

```diff
@@ -126,5 +129,10 @@ else()
 	message(FATAL_ERROR "${PROJECT_NAME} -- OpenCV -- Version-${OpenCV_VERSION_MAJOR}.${OpenCV_VERSION_MINOR}.X Not Supported")
 endif()
 
+# rocAL Support
+include_directories(${ROCM_PATH}/libexec/mivisionx/rali/python/)
```

Comment:
I am not sure if this is the correct lib path for 5.3 ROCm release. I thought it is ${ROCM_PATH}/lib